test/luatest_helpers/proxy/connection.lua

```diff
@@ -0,0 +1,141 @@
+local checks = require('checks')
```

Comment:
Being protocol aware will be added in future (at least I plan to do it). But right now there are no resources to do it.
As for input-output naming - how about changing it to "client socket" and "server socket"?

Reply:
Sounds good then. As for the naming - client/server terms are good for the server itself, because peer and server sockets are very different. But in `Connection` the client/server socket difference makes little sense IMO - they are equal bidirectional data-sockets, could be just `peer1` and `peer2`. Current naming is also good.